docs/book/src/quick-start.md

```diff
@@ -237,8 +238,8 @@ spec:
     - image: ghcr.io/azure/azure-workload-identity/msal-go
       name: oidc
       env:
-      - name: KEYVAULT_NAME
-        value: ${KEYVAULT_NAME}
+      - name: KEYVAULT_URL
+        value: ${KEYVAULT_URL}
```

Comment:
Let's add `KEYVAULT_URL` as an additional environment variable and not remove `KEYVAULT_NAME`. If old version of `msal-go` is used, it'll not work with `KEYVAULT_URL`. We can remove `KEYVAULT_NAME` after a few releases and after every [example](https://github.com/Azure/azure-workload-identity/tree/main/examples) is updated to use `KEYVAULT_URL`.

Reply:
updated.